node/src/manager/deployment.rs

```diff
@@ -228,6 +229,10 @@ impl Deployment {
             if let Some(status) = status {
                 let chain = &status.chains[0];
                 rows.extend(vec![
+                    status
+                        .paused
+                        .map(|b| b.to_string())
+                        .unwrap_or("---".to_string()),
```

Comment:
What is this?

Reply:
I still dont understand why --- rather than defaulting to a sensible value like "false"?

Reply:
I'm approving to unblock but I'd rather change this, however I'll leave that decision to you

Reply:
Its a placeholder for `None` the value is a `Option<bool>`